src/index.ts

```diff
@@ -13,8 +13,28 @@ Ensure the web UI is hosted by a Holochain Conductor or manually specify url as
   const ws = new Client(url)
   ws.on('open', () => {
     const call = (...methodSegments) => (params) => {
-      const method = methodSegments.length === 1 ? methodSegments[0] : methodSegments.join('/')
-      return ws.call(method, params)
+      let instanceId
+      let zome
+      let func
+
+      if (methodSegments.length === 1) {
+        [instanceId, zome, func] = methodSegments[0].split('/')
```

Comment:
What about `info/instances` or any other interface method? We may want a more "raw" version of call too. One that calls `call`, and one that calls anything else.

Reply:
Yeah I totally forgot about this. I think we need a `callZome` helper function and the original `call` function

Reply:
Whatcha think? It does mean there is an API change now but since not many people use this yet they can slowly migrate to using callZome before the original call syntax is deprecated.